app.js

```diff
@@ -292,4 +289,25 @@ function formatRedirectPath(path) {
     return path;
 }
 
+// Load default graph available in the "ttlUrl" key
+// if no graph is already available in the Virtuoso
+async function loadDefaultGraphs() {
+    const sources = await sourceModel.getAllSources();
+    const graphs = await rdfDataModel.getGraphs();
```

Comment:
entourer le `getGraph` dans un `try catch` pour ne pas que l'application crash quand il n'y a pas de triplestore de dispo (c'est pour ca que le test `build-and-run` échoue